phidgets_api/src/motor.cpp

```diff
@@ -155,15 +171,24 @@ void Motor::setAcceleration(double acceleration) const
     }
 }
 
+bool Motor::backEMFSensingSupported() const
+{
+    return back_emf_sensing_supported_;
+}
+
 double Motor::getBackEMF() const
 {
-    double backemf;
-    PhidgetReturnCode ret = PhidgetDCMotor_getBackEMF(motor_handle_, &backemf);
-    if (ret != EPHIDGET_OK)
+    double backemf = 0.0;
+    if (back_emf_sensing_supported_)
```

Comment:
I think we should put this code back to how it was, and instead just add at the top:

```
if (!back_emf_sensing_supported_)
{
    throw Phidget22Error("Back EMF not supported", EPHIDGET_UNSUPPORTED);
}
```